tests/validation/tests/v3_api/test_hosted_eks_cluster.py

```diff
@@ -99,6 +101,44 @@ def test_create_hosted_eks_cluster_2():
     create_and_validate_eks_cluster(cluster_config)
 
 
+@ekscredential
+def test_eks_v2_create_import_cluster():
+    ec2_cloud_credential = get_aws_cloud_credential()
+    display_name = create_resources_eks()
+    cluster_name = random_test_name("test-auto-eks")
+
+    eks_config_temp = {
+        "amazonCredentialSecret": ec2_cloud_credential.id,
+        "displayName": display_name,
+        "imported": True,
+        "privateAccess": False,
+        "publicAccess": False,
+        "region": EKS_REGION,
+        "secretsEncryption": False,
+        "type": "eksclusterconfigspec"
+    }
+
+    cluster_config = {
+        "eksConfig": eks_config_temp,
+        "name": cluster_name,
+        "type": "cluster",
+        "dockerRootDir": "/var/lib/docker",
+        "enableNetworkPolicy": False,
+        "enableClusterAlerting": False,
+        "enableClusterMonitoring": False
+    }
+    create_and_validate_eks_cluster(cluster_config,
+                                    imported=True)
+    AmazonWebServices().delete_eks_cluster(cluster_name=display_name)
```

Comment:
Delete the cluster in a finalizer so if `create_and_validate_eks_cluster` fails after the cluster is created, it still gets cleaned up.

Reply:
Addressed in changes. Cluster(s) are deleted in finalizer of `create_project_client`.